initializr-service/src/main/resources/application.yml

```diff
@@ -1256,6 +1263,17 @@ initializr:
           links:
            - rel: reference
              href: https://docs.spring.io/spring-shell/docs/2.0.0.M2/reference/htmlsingle/
+        - name: Spring Statemachine
```

Comment:
We don't use the term "Spring" anywhere else in names except for Spring Shell. Perhaps we could drop the spring part for consistency?